src/js/constans.js

```diff
@@ -1,2 +1,5 @@
 export const {API_KEY} = process.env;
-export const http = 'https://api.themoviedb.org/3';
+export const API_URL = 'https://api.themoviedb.org/3/movie/';
+export const API_SEARCH = 'https://api.themoviedb.org/3/search/movie';
+export const IMG_URL = 'https://image.tmdb.org/t/p/original';
+export const root = document.getElementById('root');
```

Comment:
Better way to use uppercase letters for constants. 
`export const ROOT = document.getElementById('root');`